capsules/src/ieee802154/driver.rs

```diff
@@ -0,0 +1,827 @@
+//! Implements a userspace interface for sending and receiving IEEE 802.15.4
```

Comment:
This should have a short description, then a blank comment line, then this paragraph to show up better on `rustdoc`.

Reply:
Cool, will do.